gate-core/src/main/groovy/com/netflix/spinnaker/gate/model/manageddelivery/ConstraintState.java

```diff
@@ -27,9 +26,9 @@
   String artifactVersion;
   String type;
   String status;
-  Instant createdAt;
+  String createdAt;
   String judgedBy;
-  Instant judgedAt;
+  String judgedAt;
```

Comment:
I was very confused and wondering how this compiles until I realized this is a gate, not a keel PR 🤣 I think we decided not to strongly type MD things in gate after I originally added this class, so feel free to delete entirely in favor of using a Map if you're inclined.

Reply:
I felt like there wasn't an immediate need to remove all the types from this class. But I might do that in the future if we're iterating on this class a bunch in `keel`